install.sh

```diff
@@ -59,9 +59,25 @@ case $PATH in
         ;;
 esac
 
-printf "Install location [default: %s]: " "$default_bin"
-read -r bindir < /dev/tty
-bindir=${bindir:-$default_bin}
+while [[ $# -gt 0 ]]
```

Comment:
Not all shells have `[[`, but `[` should work:
```suggestion
while [ $# -gt 0 ]
```

Reply:
FYI, you can use https://www.shellcheck.net/ to lint shell scripts

```
[Line 62:](javascript:setPosition(62, 7))
while [[ $# -gt 0 ]]
      ^-- [SC3010](https://www.shellcheck.net/wiki/SC3010) (warning): In POSIX sh, [[ ]] is undefined.
 
[Line 74:](javascript:setPosition(74, 4))
if [[ -z "$install_dir" ]]; then
   ^-- [SC3010](https://www.shellcheck.net/wiki/SC3010) (warning): In POSIX sh, [[ ]] is undefined.
```